media/livestream/src/create_channel.php

```diff
@@ -0,0 +1,145 @@
+ *                                    "gs://my-bucket/my-output-folder/")
+ */
+function create_channel(
+    string $callingProjectId,
+    string $location,
+    string $channelId,
+    string $inputId,
+    string $outputUri
+): void {
+    // Instantiate a client.
+    $livestreamClient = new LivestreamServiceClient();
+
+    $parent = $livestreamClient->locationName($callingProjectId, $location);
+    $channelName = $livestreamClient->channelName($callingProjectId, $location, $channelId);
+    $inputName = $livestreamClient->inputName($callingProjectId, $location, $inputId);
+
+    $channel =
+        (new Channel())
+            ->setName($channelName)
+            ->setInputAttachments([
+                (new InputAttachment())
+                    ->setKey('my-input')
+                    ->setInput($inputName)
+            ])
+            ->setElementaryStreams([
+                (new ElementaryStream())
+                    ->setKey('es_video')
+                    ->setVideoStream(
+                        (new VideoStream())
+                            ->setH264(
+                                (new VideoStream\H264CodecSettings())
+                                    ->setProfile('high')
+                                    ->setWidthPixels(1280)
+                                    ->setHeightPixels(720)
+                                    ->setBitrateBps(3000000)
+                                    ->setFrameRate(30)
+                            )
+                    ),
+                (new ElementaryStream())
+                    ->setKey('es_audio')
+                    ->setAudioStream(
+                        (new AudioStream())
+                            ->setCodec('aac')
+                            ->setChannelCount(2)
+                            ->setBitrateBps(160000)
+                    )
+            ])
+            ->setOutput(
+                (new Channel\Output())
+                                ->setUri($outputUri)
+            )
+            ->setMuxStreams([
+            (new MuxStream())
+                ->setKey('mux_video')
+                ->setElementaryStreams(['es_video'])
+                ->setSegmentSettings(
+                    (new SegmentSettings())
+                        ->setSegmentDuration(
+                            (new Duration())
+                                ->setSeconds(2)
+                        )
+                ),
+            (new MuxStream())
+                ->setKey('mux_audio')
+                ->setElementaryStreams(['es_audio'])
+                ->setSegmentSettings(
+                    (new SegmentSettings())
+                        ->setSegmentDuration(
+                            (new Duration())
+                                ->setSeconds(2)
+                        )
+                )
+            ])
+            ->setManifests([
+                (new Manifest())
+                    ->setFileName('manifest.m3u8')
+                    ->setType(1)
```

Comment:
Let's use the constant here so it's clearer:

```suggestion
                    ->setType(ManifestType::HLS)
```

Reply:
Updated style